unix-training-notebooks/Answers.ipynb

```diff
@@ -182,23 +182,27 @@
     "  \n",
     "`awk -F\"\\t\" '$4 ~ /gene/ {print $4}' exercises.bed | sort -u | wc -l`  \n",
     "  \n",
-    "Alternatively, the names can be found like this:  \n",
+    "Counting how many times each of the unique names appear can be found like this:  \n",
```

Comment:
I think the edit went a bit astray here?   "counting how many... can be found like" does read very well.

possibly something like...
> Occurrences of unique names can be counted like this:
?